start_docs_server.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# convient script to run the docs server. It automatically rebuilds and restarts when you change code. You just need to refresh your browser. 
+WATCH_PATH="."
```

Comment:
Maybe wanna add `cd "$(dirname "$0")"` to the top to make sure the script is running from where you expect it to.  If people say "../../start_docs_server.sh" for example this wouldn't work super well.

Reply:
good idea